docs/pages/enroll-resources/agents/azure.mdx

```diff
@@ -31,17 +31,15 @@ your Azure joining token.
 - An Azure Virtual Machine running Linux with the Teleport binary installed. The
   Virtual Machine must have a [Managed
   Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
-  assigned to it with permission to read virtual machine info.
+  assigned to it.
 - (!docs/pages/includes/tctl.mdx!)
 
 ## Step 1/5. Set up a Managed Identity
 
 Every virtual machine hosting a Teleport process using the Azure method to join
-your Teleport cluster needs a Managed Identity assigned to it. The identity
-requires the `Microsoft.Compute/virtualMachines/read` permission so Teleport can
-look up the virtual machine. No other permissions are required.
+your Teleport cluster needs a Managed Identity assigned to it.
 
-(!docs/pages/includes/server-access/azure-join-managed-identity.mdx!)
+(!docs/pages/includes/server-access/azure-join-system-assigned-identity.mdx!)
```

Comment:
Would it make sense to inline the partial here since there's only one guide that includes it?